lib/src/layer/polygon_layer/polygon_layer.dart

```diff
@@ -76,15 +82,17 @@ class PolygonLayer<R extends Object> extends StatefulWidget {
     super.key,
     required this.polygons,
     this.useAltRendering = false,
+    bool debugAltRenderer = false,
     this.polygonCulling = true,
     this.simplificationTolerance = 0.5,
     this.polygonLabels = true,
     this.drawLabelsLast = false,
     this.hitNotifier,
-  }) : assert(
+  })  : assert(
           simplificationTolerance >= 0,
           'simplificationTolerance cannot be negative: $simplificationTolerance',
-        );
+        ),
+        debugAltRenderer = kDebugMode && debugAltRenderer;
```

Comment:
What is the reason that the `debugAltRenderer` gets limited to the debug mode and can't be used otherwise? It could still be useful in production with e.g. a debug/dev menu.

If users want to have it enabled just in dev mode they could use kDebugMode themselfs and have full control.